### DISCUSSION TITLE
Ability to play an "unmatched file"

### DISCUSSION THREAD

**Discussion by FelixSchwarz** [Dusk-Labs/dim#601] [Q&A]
I wanted to use dim to share family videos + other private recordings which are obviously neither a known movie nor a show. It seems to me as if each media item in dim must be "matched".
Is there a way of just importing some private videos? Can I somehow add the metadata in a custom file?

This seems to be a pretty basic question but I did not find any docs regarding that so far.

**Comment by mhagnumdw** [comment]
I would also like this functionality.
